css/styles.layout.css

```
@@ -2901,6 +3117,32 @@
+        white-space: nowrap;
+        box-shadow: var(--shadow);
+      }
+      .rerun-timeline-tooltip-name {
+        font-weight: 600;
+        color: var(--text);
+      }
+      .rerun-timeline-tooltip-meta {
+        color: var(--muted);
+        font-size: 10.5px;
+      }
+      }
```

Stylelint (17.9.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)